### PR TITLE
01

### PR DESCRIPTION
**Pull request by mvala**

### CODE REVIEW

File: scripts/make.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PSD_DIR="$(dirname $(dirname $(readlink -m $0)))"
```
Comment (lhusova) [inline scripts/make.sh]: What is done in lines 3, 5 and 8

Comment (mvala) [inline scripts/make.sh]: With ```readlink -m $0``` we are getting full path of script we are running. eg. ```<something>/scripts/make.sh``` with ```dirname``` one get directory name. when we do ```dirname``` 2 times we get project directory

File: scripts/make.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+[ -d $PSD_DIR/build ] ||  mkdir $PSD_DIR/build
```
Comment (mvala) [inline scripts/make.sh]: We are checking if directory ```$PSD_DIR/build``` exists. If not (```||```) we create it

File: scripts/make.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+cd $PSD_DIR/build || exit 1
+[ -f Makefile ] || cmake ../
```
Comment (mvala) [inline scripts/make.sh]: We check if ```Makefile``` exists. If not we do ```cmake ../```

File: scripts/make.sh (hunk 2 above)
Comment (mvala) [inline scripts/make.sh]: @lhusova

File: scripts/make.sh (hunk 3 above)
Comment (mvala) [inline scripts/make.sh]: @lhusova

**Pull request merged**